test/models/pregnancy_test.rb

```diff
@@ -43,4 +43,23 @@ class PregnancyTest < ActiveSupport::TestCase
       assert @pregnancy.created_by
     end
   end
+
+  before do
+    @pregnancy.dcaf_soft_pledge = 500
+    @pt_1.appointment_date = DateTime.now + 14
+    @clinic = create :clinic
```

Comment:
Same deal as above -- go ahead and set `@pt_1.clinic_name` to some string or another. Doesn't matter what just as long as it's not nil.